app/src/main/java/org/mozilla/fenix/search/toolbar/ToolbarView.kt

```diff
@@ -32,7 +32,7 @@ interface ToolbarInteractor {
      * Called when a user hits the return key while [ToolbarView] has focus.
      * @param url the text inside the [ToolbarView] when committed
      */
-    fun onUrlCommitted(url: String)
+    fun onUrlCommitted(url: String, fromHomeScreen: Boolean = false)
```

Comment:
Let's add the @param fromHomeScreen here since we already have it defined in the kDoc.